src/test/ui/const-generics/concrete-const-impl-static-method.rs

```diff
@@ -0,0 +1,17 @@
+// Test that a static method within an impl block of concrete const type i.e. A<2>, is callable.
```

Comment:
```suggestion
// Test that an associated non-method within an impl block of concrete const type i.e. A<2>, is callable.
```
(also make consequence changes where needed including the file name)

Reply:
Done